scrape/target.go

```diff
@@ -390,6 +427,34 @@ func populateLabels(lset labels.Labels, cfg *config.ScrapeConfig) (res, orig lab
 		return nil, nil, err
 	}
 
+	var interval string
```

Comment:
Can we do something to avois this code duplication, or does it end being more complex?

Reply:
Over here we do more than just parsing, we also check to make sure the value is valid, so probably not.